microsoft-edge/webview2/index.md

```diff
@@ -20,6 +20,12 @@ With WebView2, you can embed web code in different parts of your native app, or
 To start building a WebView2 app, see [Get started with WebView2](get-started/get-started.md).
 
 
+<!-- ------------------------------ -->
+#### Video: Ignite | March 2021 | Building, Deploying, and Managing WebView2 Applications
+
+[![Thumbnail image for video "Building, Deploying, and Managing WebView2 Applications"](./index-images/build-deploy-wv2-apps.png)](https://www.youtube.com/watch?v=LX-eXvcSx6c)
```

Comment:
I don't think we should add this video here. I would prefer if we moved it further down to the bottom, in a see also section instead.
The reason is that it's most probably very outdated by more recent WebView2 developments, and no longer represents an up to date intro to people wanting to learn more about WebView2.

Reply:
done here & regular article